webapp/src/components/post_menu_action/create_issue/create_issue.jsx

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+
+import {id as pluginId} from '../../../manifest';
+
+export default class CreateIssuePostMenuAction extends PureComponent {
+    static propTypes = {
+        isSystemMessage: PropTypes.bool.isRequired,
+        open: PropTypes.func.isRequired,
+        postId: PropTypes.string,
+        connected: PropTypes.bool.isRequired,
+    };
+
+    static defaultTypes = {
+        locale: 'en',
+    };
+
+    handleClick = (e) => {
+        const {open, postId} = this.props;
+        e.preventDefault();
+        open(postId);
+    };
+
+    connectClick = () => {
+        window.open('/plugins/' + pluginId + '/user/connect', '_blank');
+    };
+
+    render() {
+        if (this.props.isSystemMessage || !this.props.connected) {
+            return null;
+        }
+
+        const content = (
+            <button
+                className='style--none'
+                role='presentation'
+                onClick={this.handleClick}
+            >
+                <i className='fa fa-github fa-lg'/>&nbsp;
```

Comment:
Could we handle this space with some ccs styling of the icon?

Reply:
I am not sure what to do here, can you help? Thanks

Reply:
This has been replaced with the GitHubIcon Component and now matches the `attach to issue` style.